test/integration/integration_test.go

```diff
@@ -214,6 +240,9 @@ func TestMain(m *testing.M) {
 			"vault_token":   os.Getenv("VAULT_TOKEN"),
 		},
 	})
+	if isScaleTest {
+		tfOptions.Vars["cluster_name"] = clusterName
+	}
```

Comment:
I think we could put `["cluster_name"] = clusterName` as part of the `tfOptions` [instantiation block](https://github.com/hashicorp/vault-secrets-operator/pull/916/files#diff-ca62391b26e30e01d1f8cee0eac7829ba8e9b5e22d80eab0e83881086fb4177bR234-R240) for cleanliness.

At the moment, only `providers/eks.tf` uses `cluster_name`. If `providers/kinds.tf` needs to use `cluster_name`, we could update `clusterName`